doc/api/process.md

```diff
@@ -1917,6 +1917,44 @@ console.log('After:', getActiveResourcesInfo());
 //   After: [ 'TTYWrap', 'TTYWrap', 'TTYWrap', 'Timeout' ]
 ```
 
+## `process.getBuiltin(id)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `id` {string} ID of the built-in module being requested. Must not contain the `node:` prefix.
+* Returns: {Object}
+
+`process.getBuiltin(id)` provides a way to load the built-in modules
+in a globally available function. ES Modules that need to support
+other environments can use it to conditionally load a Node.js builtin
+when it is run in Node.js, without having to deal with the resolution
+error that can be thrown by `import 'node:id'` in a non-Node.js
+environment or having to use dynamic `import()` which either turns the
+module into an asynchronous module, or turns a synchronous API into an
+asynchronous one.
+
+```mjs
+if (globalThis.process && globalThis.process.getBuiltin) {
+  // Run in Node.js, use the Node.js fs module.
+  const fs = globalThis.process.getBuiltin('fs');
+  // If `require()` is needed to load user-modules, use createRequire()
+  const m = globalThis.process.getBuiltin('module');
+  const require = m.createRequire(import.meta.url);
+  const foo = require('foo');
+}
+```
+
+If `id` specifies a built-in available in the current Node.js process,
+`process.getBuiltin()` method returns the corresponding built-in
+module, which is the same as the object returned by
+[`require(id)`][`require()`]. If `id` does not correspond to any
```

Comment:
Wouldn't it be more accurate to say it's the same as ``require(`node:${id}`)`` (i.e. it bypasses the `require` cache)
```suggestion
module, which is the same as the object returned by
[``require(`node:${id}`)``][`require()`]. If `id` does not correspond to any
```

https://github.com/nodejs/node/blob/4d59a9deda7c624d3341e3ee37452aee960f8816/doc/api/modules.md#L402-L405